skein/utils.py

```diff
@@ -82,12 +84,15 @@ def grpc_fork_support_disabled():
     gRPC (>= 1.15) results in extraneous error logs currently. For now we
     explicitly disable fork support for gRPC clients we create.
     """
-    key = 'GRPC_ENABLE_FORK_SUPPORT'
-    try:
-        os.environ[key] = '0'
-        yield
-    finally:
-        del os.environ[key]
+    if version.parse(grpcio.__version__) < version.parse('1.18.0'):
```

Comment:
I'd use `distutils.version.LooseVersion` instead, as it's part of the standard library:

```
if LooseVersion(grpcio.__version__) < '1.18.0':
    ...
```

Reply:
I agree 👍